PingProtect/PingProtect/Sources/Callbacks/PingOneProtectEvaluationCallback.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  PingOneProtectEvaluationCallback.swift
+//  PingProtect
+//
+//  Copyright (c) 2024 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+import Foundation
+import FRAuth
+
+
+/**
+ * Callback to evaluate Ping One Protect
+ */
+open class PingOneProtectEvaluationCallback: MultipleValuesCallback {
```

Comment:
@witrisna: I know this has nothing to do with this PR itself, but is it too late to discuss the name of this callback? I just noticed that it has `PingOne*`, rather than just `Ping*` (referring back to our naming discussion). Maybe `PingProtectCollectCallback` and `PingProtectInitCallback` to keep it a bit shorter?

Reply:
The naming comes agreed with other Team Trust Network, AM, kind of late for name changing now.